lxd/warnings.go

```diff
@@ -32,16 +33,16 @@ import (
 var warningsCmd = APIEndpoint{
 	Path: "warnings",
 
-	Get: APIEndpointAction{Handler: warningsGet},
+	Get: APIEndpointAction{Handler: warningsGet, AccessHandler: allowPermission(auth.ObjectTypeServer, auth.EntitlementCanEdit)},
```

Comment:
Shouldn't this be `auth.EntitlementCanView`?

Reply:
`can_view` on `server` is used for the `/1.0` endpoint so anyone is allowed. I wasn't sure if warnings might contain any sensitive information (it's currently admin only) so I kept it at `server:can_edit`.

Reply:
Just to clarify. For untrusted users the behaviour is the same for `/1.0`. Any authenticated user can see the contents of `lxc info` **without** the server config. Only users with `can_edit` on `server` can see server config.